thirdparty/mbedtls/library/net_sockets.c

```diff
@@ -49,7 +49,7 @@
 #undef _WIN32_WINNT
 #endif
 /* Enables getaddrinfo() & Co */
-#define _WIN32_WINNT 0x0501
+#define _WIN32_WINNT 0x0601
```

Comment:
Any chance to get this fixed upstream? It's bogus of them to just override `_WIN32_WINNT` with whatever they need as minimum. They should do something like:
```
#if !defined(_WIN32_WINNT) || (_WIN32_WINNT < 0x0501)
/* Enables getaddrinfo() & Co */
#define _WIN32_WINNT 0x0501
#endif
```